client/crypto-sdk-android/crypto-sdk/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ cmake_minimum_required(VERSION 3.6)
 set(CMAKE_CXX_STANDARD 11)
 
 file(GLOB CRYPTO_SDK_ANDROID_SRC
-  "src/main/cpp/*.h"
+  "src/main/cpp/*.h"§
```

Comment:
`§`?

Reply:
i guess only me and god know what this was, and since i seem to have forgotten, you should ask him.. fixing